src/lib/gravityErrorHandler.ts

```diff
@@ -53,50 +57,53 @@ export const formatGravityError = (error) => {
       return {
         type: "error",
         error: freeBody,
-        statusCode: error?.statusCode,
+        statusCode: error.statusCode,
         message: parsedError.error || parsedError.message,
       }
-    } else if (freeBody.error as string) {
+    } else if (freeBody.error) {
       return {
         type: "error",
         message: freeBody.error,
+        statusCode: error.statusCode,
       }
-    } else if (freeBody as string) {
+    } else if (typeof freeBody === "string") {
```

Comment:
If `freeBody` is a string, and is not JSON, there's not much we can do

Reply:
**question (non-blocking):** And in this case, we can't assume the parent `error` has a `statusCode`? (Noticing that this seems to be the only branch that doesn't include that in the return value)

Reply:
Oh yea! We can throw in `statusCode: error.statusCode` everywhere for completeness as it will be present for http errors, and not otherwise, and it is defined as nullable [here](https://github.com/artsy/metaphysics/blob/728647dd58cb2d9f0c3de146b2576391bf5fd41c/src/lib/gravityErrorHandler.ts#L34)